src/components/Home/Home.js

```diff
@@ -1,24 +1,47 @@
-import { useTheme } from '@react-navigation/native';
-import React from 'react';
-import { Text, View } from 'react-native';
-import { useSelector } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { View, Image, FlatList } from 'react-native';
+import useFetchMovies from '../../hooks/useFetchMovies';
+import Center from '../Center';
+import { getConfiguration } from '../../helpers/MoviesHelper';
+
+import logoIcon from 'assets/ic_logo/ic_logo.png';
 import styles from 'components/Home/styles';
-import TextStyles from 'helpers/TextStyles';
-import strings from 'localization';
-import { getUser } from 'selectors/UserSelectors';
+import MoviesListItem from 'components/MoviesListItem';
 
 function Home() {
-  const { colors } = useTheme();
-  const user = useSelector(getUser);
+  const [urlPrefix, setUrlPrefix] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [movies, fetchMore] = useFetchMovies();
+
+  useEffect(() => {
+    setIsLoading(true);
+    const fetchCfg = async () => {
+      const response = await getConfiguration();
+      setUrlPrefix(
+        response.images.base_url + response.images.backdrop_sizes[1]
+      );
+    };
```

Comment:
Does this needs to be inside the useEffect?

Reply:
I think so, this way it will only fetch when the component mounts, otherwise it will trigger an infinite loop. But maybe I'm missing something?

Reply:
My idea was to move the async method out of the useEffect with a useCallback so you don't have an infinite render loop to make this cleaner

Reply:
Gotcha, yes, I can definitely change it to that